lib/src/main/java/com/jesusm/holocircleseekbar/lib/HoloCircleSeekBar.java

```diff
@@ -447,6 +434,46 @@ public void setValue(float newValue) {
         }
     }
 
+    /**
```

Comment:
These methods also respect current indentation spaces. I've seen that `setValue` doesn't respect it so if you can it'd be great if you can fix that.